Dockerfile

```diff
@@ -149,6 +150,15 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
         SDK_MANAGER_PROXY_OPTIONS="--proxy=http --proxy_host=${PROXY_HOST_AND_PORT%:*} --proxy_port=${PROXY_HOST_AND_PORT##*:}"; \
     fi && \
     yes | $ANDROID_HOME/cmdline-tools/bin/sdkmanager $SDK_MANAGER_PROXY_OPTIONS --sdk_root=$ANDROID_HOME "platform-tools" && \
+    # Install 'CocoaPods'. As https://github.com/CocoaPods/CocoaPods/pull/10609 is needed but not yet released, make a
+    # build from the latest revision of the master branch.
+    git clone https://github.com/CocoaPods/CocoaPods.git && \
+        cd CocoaPods && \
+        git checkout 9461b346aeb8cba6df71fd4e71661688138ec21b && \
```

Comment:
The same idea from above applies here to (hopefully) improve download performance.

Reply:
I'd like to leave this improvement up to a follow-up PR as this does cause me quite a bit: context switch and testing time. Ok?